node/tests/GlideClient.test.ts

```diff
@@ -1331,8 +1333,14 @@ describe("GlideClient", () => {
 
             // transaction test
             const transaction = new Transaction()
-                .hset(hashPrefix + 1, { name: "Alice", age: "30" })
-                .hset(hashPrefix + 2, { name: "Bob", age: "25" })
```

Comment:
Note: 1st hset with HashDataType as input and 2nd hset call using the Record type as input.